src/app/shared/add-user/add-user.service.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { UserInfoService } from '@hypertrace/common';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class AddUserService {
```

Comment:
This file is temporary. Since the `addUser()` method interacts with the `/user-preferences` endpoint, this method should ideally be added to the [UserPreferenceService][1] from #56. Will add the change when the service is available in `rzp_main`.

[1]: https://github.com/razorpay/hypertrace-ui/pull/56/files#diff-d0ef16a7e639d1c040567b66d3c9f391f6543c671838fe8689f6dfc43d8f8292R13

Reply:
✅ File deleted